### PR TITLE
add openshift-e2e-vsphere-ovn-dualstack workflow

### PR DESCRIPTION
**Pull request by rbbratta**

### CODE REVIEW

**Comment by rbbratta** [comment]
This doesn't work because we don't select the `vsphere-ibm-7-quota-slice`

**Comment by rbbratta** [comment]
PTAL @zhaozhanqi

File: core-services/ci-chat-bot/workflows-config.yaml [hunk 1]
```diff
@@ -257,6 +257,8 @@ workflows:
     platform: openstack
   openshift-e2e-vsphere:
     platform: vsphere
+  openshift-e2e-vsphere-ovn-dualstack:
```
Comment (liangxia) [inline core-services/ci-chat-bot/workflows-config.yaml:260]: Is it the right one you want, did you mean to add `cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack` ?

Comment (rbbratta) [reply core-services/ci-chat-bot/workflows-config.yaml:260]: ` cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack` is already added.  We want both eventually.  Everyone should be testing dualstack, it's the future.

Comment (liangxia) [reply core-services/ci-chat-bot/workflows-config.yaml:260]: Ack

**Pull request merged**